### DISCUSSION TITLE
multiplicity cuts

### DISCUSSION THREAD

**Discussion by Purnath** [MadAnalysis/madanalysis5#267] [Q&A] [answered]
Dear ma5 team,

i have muon and a tau in the final state
 tau decays, now it can go to any hadronic or leptonic decay i am using pythia ,delphes and madanalysis5 in the mg5 environment
i want only the hadronic decay of the tau
for that i want only one muon and zero electron in the final state i want to know how to implement this cut ?
i tried implementing these cuts in madanalysis5 using
select (mu) N==1
select (e) N==0

but i'm not getting the results
 i want number of events after the cuts

best regards
Purnath

**Comment by BFuks** [comment, marked as answer]
Hi Purnath,

Please check our reference cards (the `doc` folder in MA5)… This is all included in there:
  select N(mu)==1
  select N(e) ==0

Cheers,

Benjamin
